src/TLuaInterpreter.cpp

```diff
@@ -10401,6 +10401,17 @@ int TLuaInterpreter::getDiscordParty(lua_State* L)
     return 2;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#resetDiscordData
+int TLuaInterpreter::resetDiscordData(lua_State* L)
+{
+    mudlet* pMudlet = mudlet::self();
+    auto& host = getHostFromLua(L);
+
+    pMudlet->mDiscord.resetData(&host);
```

Comment:
```suggestion
    auto& host = getHostFromLua(L);
    mudlet::self()->mDiscord.resetData(&host);
```

Would this simplification work?